training/training_population.py

```diff
@@ -0,0 +1,29 @@
+from utils import text_extractor
+SIZE_OF_TRAINING_EXAMPLE = 5
+
+
+class WebsiteIterator:
+    def __init__(self, url):
```

Comment:
this is a classic case where we need to test for logic.
this will be your "proof" you tested many edge cases.